app/controllers/admin/users_controller.rb

```diff
@@ -10,6 +10,7 @@ def index
 
   # GET /users/1
   def show
+    @displayed_attributes = @user.attributes.keys.select{|k| k != 'encrypted_password'}
```

Comment:
Here you can use `reject` instead of inverting the `select`
`@user.attributes.keys.reject{ |k| k == 'encrypted_password' }`